plugins/main/bridge/bridge.go

```diff
@@ -358,6 +364,15 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
+	// release IP in case of failure
+	defer func(ok *bool) {
```

Comment:
Small question - why it's passed there as a pointer value? Inside of this func it's already accessible as `success` and it will have a correct value in time of deferred execution, so there is no need for this `ok` variable as `success` can be used directly.

Reply:
You're right. Fixed.